src/test/java/APITest/register_sucess.java

```diff
@@ -0,0 +1,55 @@
+package APITest;
+
+import static org.hamcrest.Matchers.*;
+
+import io.restassured.path.json.JsonPath;
+
+import static io.restassured.RestAssured.*;
+
+public class register_sucess {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		
+    //Register Successful
+	String response= given().spec(Utility.requestbuild()) 
+		             .body(Payload.register())
+                     .when().post("/api/register")
+                     .then().spec(Utility.responsebuild()).assertThat().body("id",equalTo(4)).extract().response().asString();
```

Comment:
Avoid long lines as they are hard to read. Again something your auto-formatter will take care of.